### ISSUE TITLE
AllOpsResolver is missing the Cast operation by default

### ISSUE THREAD

**Issue by ScholliYT** [tensorflow/tflite-micro#861]
I noticed that the AllOpsResolver is currently missing the Cast operation. The operation is already implemented in TFLM and can be used by adding it to the Resolver manually by `AddCast()`. However, it is not part of the default operations used by the `AllOpsResolver`.

```python
@tf.function(input_signature=[tf.TensorSpec(shape=[1], dtype=tf.float32)])
def cast_example(x):  
  y = tf.cast(x, dtype=tf.int32)
  return y

input_data = tf.constant([1.42])
y = cast_example(input_data)
print(y, y.shape)

converter = tf.lite.TFLiteConverter.from_concrete_functions([cast_example.get_concrete_function()], cast_example)

#converter.allow_custom_ops = True
tflite_model = converter.convert()

TFLITE_FILE_PATH = 'model.tflite'
# Save the model.
with open(TFLITE_FILE_PATH, 'wb') as f:
  f.write(tflite_model)
```

Trying to use that model in TFLM with the `AllOpsResolver` currently results in the following error message:
```
Didn't find op for builtin opcode 'CAST' version '1'. An older version of this builtin might be supported. Are you using an old TFLite binary with a newer model?

Failed to get registration from op code CAST, with opcode_index 0
```

Only after manually adding the Cast operation to the resolver the above model can be executed with TFLM. 
```cpp
static tflite::AllOpsResolver resolver;
resolver.AddCast();
```

**Comment by ScholliYT** [comment]
This can be closed as #862 is merged now
